src/app/components/board/board.component.ts

```diff
@@ -13,7 +12,7 @@ export class BoardComponent implements OnInit {
 
   ngOnInit() {
     this.board = BoardData[2020];
-    console.log(this.board);
+    //console.log(this.board);
     AOS.init();
   }
```

Comment:
@Shehanka You need to handle onClick event to fetch the board details year wise.

Reply:
okay! I'll